soc/riscv/riscv-privilege/andes_v5/ae350/linker.ld

```diff
@@ -116,18 +116,10 @@ SECTIONS
     GROUP_START(ROMABLE_REGION)
     __rom_region_start = ROM_BASE;
 
-    SECTION_PROLOGUE(_VECTOR_SECTION_NAME,,)
```

Comment:
should remove `_VECTOR_SECTION_NAME` definition at L31 since it is no longer being used

Reply:
Sure, thanks!